app/main/views/drafts.py

```diff
@@ -101,8 +112,10 @@ def edit_draft_service(service_id):
 @main.route('/draft-services', methods=['GET'])
 def list_drafts():
```

Comment:
I know it's not technically part of your pull request, but I'm wondering why this method isn't called `list_draft_services`.
All of the other method names have the word "service" in them somewhere.